streams/src/test/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImplTest.java

```diff
@@ -153,7 +155,7 @@ public void shouldNotHaveInvalidStoreNameOnWindowedAggregate() {
             .aggregate(MockInitializer.STRING_INIT, MockAggregator.TOSTRING_ADDER, Materialized.as(INVALID_STORE_NAME));
     }
 
-    private void doAggregateSessionWindows(final Map<Windowed<String>, KeyValue<Integer, Long>> results) {
+    private void doAggregateSessionWindows(final MockProcessorSupplier<Windowed<String>, Integer> supplier) {
```

Comment:
Some test cleanup to get rid or `ResultCollector` processor but reuse `MockProcessor` instead